src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1259,7 +1259,13 @@ format layout_optimizer::get_preferred_format(program_node& node) {
                 } catch (ov::Exception&) {
                     fmt = format::get_default_format(in_lay_rank);
                 }
-                node.set_preferred_input_fmt(i, fmt);
+
+                if (node.is_type<reshape>()) {
```

Comment:
It seems not a good idea for handle reshape separately in here. If reshape always need simple format, how about set it move to 1296 line?